Feature/spline #1017 docker generator

AdrianOlosutean opened the pull request:
fixes #1017

AdrianOlosutean committed:
spline #1017 code feedback

AdrianOlosutean committed:
spline #1017 fixed scripts

AdrianOlosutean committed:
spline #1017 updated

AdrianOlosutean committed:
spline #1017 small updates

AdrianOlosutean committed:
spline #1017 improved script

AdrianOlosutean committed:
spline #1017 small improvement

AdrianOlosutean committed:
Merge branch 'develop' into feature/spline-1017-docker-generator

dk1844 committed:
Merge branch 'develop' into feature/spline-1017-docker-generator

cerveada commented:
@dk1844 Currently the build is failing because the license is missing in some files.

dk1844 committed:
#1017 licence header added

dk1844 commented:
> @dk1844 Currently the build is failing because the license is missing in some files.

@cerveada, thanks for noticing, fixed.

dk1844 committed:
Feature/1017 data generator output name (#1102)

* #1017 
 - Custom output path, range is now 1-2@3 instead of 1-2/3 to avoid generating invalid paths
 - Using a predefined output file name for docker-tests
 - FileDispatcher file name creation externalized. Testable with e.g.
`java -jar .\target\test-data-generator-1.0.0-SNAPSHOT.jar -g triangle -r 5 -o 3 -a 3 -t testing_lineage.json`
`java -jar .\target\test-data-generator-1.0.0-SNAPSHOT.jar -g triangle -r 5 -o 3 -a 3`
